ansible/roles/ocp4-workload-dil-serverless/defaults/main.yaml

```diff
@@ -39,3 +39,73 @@ ocp4_workload_dil_serverless_webapp_operator_resource_items:
   - crd.yaml
 ocp4_workload_dil_serverless_webapp_walkthrough_locations:
   - 'https://github.com/RedHat-Middleware-Workshops/dayinthelife-serverless.git?walkthroughsFolder=/docs/labs'
+
+# Smartcity Lab
+ocp4_workload_dil_serverless_ocs_operator_namespace: openshift-storage
+ocp4_workload_dil_serverless_smartcity_edge_s3_bucket: smartcity-edge-s3-bucket
+ocp4_workload_dil_serverless_smartcity_edge_project: smartcity-edge
+ocp4_workload_dil_serverless_smartcity_central_project: smartcity-central
+ocp4_workload_dil_serverless_image_loader_job_image: quay.io/btison_smartcity/image-loader:latest
```

Comment:
Can we move this image and the other images to the `redhatintegration` organization?

Reply:
Sure, but I don't think I have access to that organization

Reply:
Send me your quay user to add you to the team.